src/Flex.php

```diff
@@ -161,11 +160,44 @@ public function update(Event $event): void
         }
 
         $this->io->writeError(sprintf('<info>Symfony operations: %d recipe%s</>', count($recipes), count($recipes) > 1 ? 's' : ''));
+        $installContribs = $this->composer->getPackage()->getExtra()['symfony']['allow-contrib'] ?? false;
         foreach ($recipes as $recipe) {
-            if ($recipe->isNotInstallable() && $recipe->isEmpty()) {
-                $this->io->writeError(sprintf('  - <warning>Ignoring</> recipe %s', $this->formatOrigin($recipe->getOrigin())));
-                $this->io->writeError('    Enable via composer config extra.symfony.allow-contrib true');
-                continue;
+            if ('install' === $recipe->getJob() && !$installContribs && $recipe->isContrib()) {
+                $this->io->writeError(sprintf('  - <warning>WARNING</> %s', $this->formatOrigin($recipe->getOrigin())));
+                $question = '    This package wants to update your project for better integration.
+    Allow?
+    [<comment>y</>] Yes
+    [<comment>n</>] No
+    [<comment>a</>] Yes for all packages, only for the current installation session
+    [<comment>p</>] Yes permanently, never ask again for this project
+    (defaults to <comment>n</>): ';
```

Comment:
Should we print a different text if the command is run in non-interactive mode?

Reply:
The question is never displayed if the command is non-interactive.

Reply:
I've made one small tweak. If non-interactive mode, it displays 'IGNORING' like before this PR and 'WARNING' in interactive mode. So, in non-interactive mode, the display is the same as before.